notebooks/250-music-generation/250-music-generation.ipynb

```diff
@@ -0,0 +1,742 @@
+{
```

Comment:
<blockquote>We use <a href="https://docs.openvino.ai/nightly/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html" target="_blank">OpenVINO Converter (OVC)</a></blockquote><p>The link points to the usage of <code>mo.convert_model</code>  from <code>openvino.tools</code> . Perhaps link should be removed until there is a page for the new API.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1242/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>done</p>